db/todo.txt

```diff
@@ -0,0 +1,8 @@
+things to do each time the schema script is baselined...
+
+remove database config settings
```

Comment:
Should this be checked in?  Why not move it to the README...

Reply:
this is just personal notes for the PR, have moved to readme anyway, as its all in one place